sklearnex/spmd/linear_model/tests/test_logistic_regression_spmd.py

```diff
@@ -116,6 +116,8 @@ def test_logistic_spmd_gold(dataframe, queue):
 @pytest.mark.parametrize("dtype", [np.float32, np.float64])
 @pytest.mark.mpi
 def test_logistic_spmd_synthetic(n_samples, n_features, C, tol, dataframe, queue, dtype):
+    # TODO: Re-enable coefficient check when oneDAL fix implemented
+    pytest.skip("Test disabled while OneDAL fix in progress")
```

Comment:
I think it is better cover it with ticket and avoid TODO commenting here.
```suggestion
    pytest.skip("Sporadic failures on coefficient check. Test disabled while OneDAL fix in progress")
```